src/scheduler/fifo.c

```diff
@@ -511,15 +504,13 @@ init_scheduling_cycle(status *policy, int pbs_sd, server_info *sinfo)
 				if (sinfo->job_formula != NULL) {
 					double threshold = policy->job_form_threshold;
 					resresv->job->formula_value = formula_evaluate(sinfo->job_formula, resresv, resresv->resreq);
-					sprintf(log_buffer, "Formula Evaluation = %.*f",
-						float_digits(resresv->job->formula_value, FLOAT_NUM_DIGITS), resresv->job->formula_value);
-					schdlog(PBSEVENT_DEBUG3, PBS_EVENTCLASS_JOB, LOG_DEBUG, resresv->name, log_buffer);
+					log_eventf(PBSEVENT_DEBUG2, PBS_EVENTCLASS_JOB, LOG_DEBUG, resresv->name, "Formula Evaluation = %.*f",
```

Comment:
This was a 'PBSEVENT_DEBUG3' right? Is this change intentional?

Reply:
Yes, it shouldn't have been DEBUG3 to begin with.  Per-job messages are at DEBUG2.  Per-job per-node messages are at DEBUG3.